airflow/providers/amazon/aws/sensors/batch.py

```diff
@@ -115,7 +115,15 @@ def execute_complete(self, context: Context, event: dict[str, Any]) -> None:
         Relies on trigger to throw an exception, otherwise it assumes execution was successful.
         """
         if event["status"] != "success":
-            raise AirflowException(f"Error while running job: {event}")
+            message = f"Error while running job: {event}"
+            # TODO: remove this if-else block when min_airflow_version is set to higher than the version that
+            # raise_failed_or_skiping_exception is introduced in BaseSensorOperator
```

Comment:
Do we know what version this will be yet? It would be easier to search if we have _Airflow X.Y_ in the comment.

Reply:
I don't, but my guess is 2.7.1?

Reply:
Let’s keep this then.

Reply:
I can send another PR after the version is decided